server/src/main/java/com/genymobile/scrcpy/AudioCapture.java

```diff
@@ -74,12 +74,12 @@ private static void startWorkaroundAndroid11() {
         Intent intent = new Intent(Intent.ACTION_MAIN);
         intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
         intent.addCategory(Intent.CATEGORY_LAUNCHER);
-        intent.setComponent(new ComponentName(FakeContext.PACKAGE_NAME, "com.android.shell.HeapDumpActivity"));
+        intent.setComponent(new ComponentName(FakeContext.getPackageNameStatic(), "com.android.shell.HeapDumpActivity"));
         ServiceManager.getActivityManager().startActivityAsUserWithFeature(intent);
     }
 
     private static void stopWorkaroundAndroid11() {
-        ServiceManager.getActivityManager().forceStopPackage(FakeContext.PACKAGE_NAME);
+        ServiceManager.getActivityManager().forceStopPackage(FakeContext.getPackageNameStatic());
```

Comment:
Same here, we want to always kill `com.android.shell` app.

Reply:
These two should be cleared up